### PR TITLE
Bundler test cases

### PR DESCRIPTION
**Pull request by Andrew-Hagedorn-ZocDoc**
@thomas-huston-zocdoc @Laura-Johannet-ZocDoc 

Everytime I go back into Bundler a couple of things bug me about the test structure:

  * The tests dont work well outside of the box of the person who wrote it so often there are failing tests
  *  Its really annoying to sprinkle a bunch of files across multiple folders
  *  You can't understand what a test is doing by reading the test itself.

This is one file of some work I did a while back in my fork of this repo.  What do you think about the test structure in `tests/integration-tests/bundler-css-spec.js`?

### CODE REVIEW

File: tests/integration-tests/bundler-css-spec.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+
+describe("Css Bundling:", function() {
+
+    var exec = require('child_process').exec,
+		fs = require('fs'),
+        testHelper = require('./helpers/integration-test-helper.js'),
```
Comment (Andrew-Hagedorn-ZocDoc) [inline tests/integration-tests/bundler-css-spec.js]: This setup section likely needs some work so its magic-ed for you. 

Same with the beforeEach and afterEach.

Comment (thomas-huston-zocdoc) [inline tests/integration-tests/bundler-css-spec.js]: Agreed, we could probably make a base `describe` class or something.

**Comment by thomas-huston-zocdoc** [comment]
This definitely seems like an improvement. Those old integration tests never passed on my machine to begin with. Hopefully these ones actually work?

**Comment by thomas-huston-zocdoc** [comment]
:+1:

**Pull request merged**